azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/service/sdk/CallingSDKEventHandler.kt

```diff
@@ -84,6 +84,16 @@ internal class CallingSDKEventHandler(
         transcriptionFeature.addOnIsTranscriptionActiveChangedListener(onTranscriptionChanged)
     }
 
+    fun onHold() {
```

Comment:
Why do we need to having this hook? 
`CallingSDKEventHandler` is used to listen the CallingSDK events. CallingSDK doesn't have extra event regarding `onHold` and `onResume`. I think we can remove them

Reply:
It's not necessary, it's just here because of consistency. when following the patterns in calling sdk wrapper.

Removed however, if we don't need now we can add later if necessary or need to publish these states.